libraries/Preferences/src/Preferences.cpp

```diff
@@ -291,7 +291,7 @@ int8_t Preferences::getChar(const char* key, const int8_t defaultValue){
     }
     esp_err_t err = nvs_get_i8(_handle, key, &value);
     if(err){
-        log_e("nvs_get_i8 fail: %s %s", key, nvs_error(err));
+        log_v("nvs_get_i8 fail: %s %s", key, nvs_error(err));
```

Comment:
why would you mute the error? it is an error, so do not mute any of them please. If you get them, then see what you are doing wrong.

Reply:
Because, if you havent saved a preference value yet, the default is assumed, that's why you provide a default. That is not a faulty situation, and not an error, a warning at most